docker_build/Dockerfile

```diff
@@ -1,77 +0,0 @@
-# Copyright (c) 2021, NVIDIA CORPORATION.
-
```

Comment:
I think this file should stay since this was added to facilitate a build inside a container without involving the use of `conda`.

Reply:
From our discussions offline I think we are good to remove this.